geoportal/vars.yaml

```diff
@@ -306,6 +306,21 @@ vars:
               unit: m
             - name: srtm
               unit: m
+          sketchStyle:
+            fill:
+              color: rgba(255, 128, 128, 0.2)
+            stroke:
+              color: rgba(255, 0, 0, 0.5)
+              lineDash: [10, 10]
+              width: 2
+            regularShape:
+              stroke:
+                color: rgba(0, 0, 0, 0.7)
+                width: 2
+              points: 4
+              radius: 8
+              radius2: 0
+              angle: 0
```

Comment:
Can't you put it in the CONST_vars.yaml?

Reply:
Is this set correctly ?
https://github.com/camptocamp/demo_geomapfish/blob/prod-2-6/geoportal/CONST_vars.yaml#L359-L377

Anyways the raster section is not default here.

Reply:
Yes, but it looks like that 
  - interfaces_config.mobile.constants.gmfMobileMeasurePointOptions
  is missing in the update_paths